### ISSUE TITLE
/bin/tar: file changed as we read it

### ISSUE THREAD

**Issue by greatghoul** [huacnlee/gobackup#10]
```
deploy@localhost:~/.gobackup$ gobackup perform
2017/10/12 15:16:45 ======== yizaoyiwan ========
2017/10/12 15:16:45 WorkDir: /tmp/gobackup/1507821405731845154/yizaoyiwan

2017/10/12 15:16:45 ------------- Databases -------------
2017/10/12 15:16:45 => database | PostgreSQL: yizaoyiwan
2017/10/12 15:16:45 -> Dumping PostgreSQL...
2017/10/12 15:16:46 dump path: /tmp/gobackup/1507821405731845154/yizaoyiwan/postgresql/yizaoyiwan/yizaoyiwan.sql
2017/10/12 15:16:46
2017/10/12 15:16:46 ------------- Databases -------------

2017/10/12 15:16:46 ------------- Archives -------------
2017/10/12 15:16:46 => includes 1 rules
2017/10/12 15:16:46 ------------- Archives -------------

2017/10/12 15:16:46 ------------ Compressor -------------
2017/10/12 15:16:46 => Compress | tgz
2017/10/12 15:16:46 [debug] /bin/tar zcf /tmp/gobackup/1507821405731845154/yizaoyiwan/2017.10.12.23.16.46.tar.gz yizaoyiwan
2017/10/12 15:16:46 /bin/tar: yizaoyiwan: file changed as we read it

2017/10/12 15:16:46 Cleanup temp dir...

2017/10/12 15:16:46 ======= End yizaoyiwan =======
```

> 2017/10/12 15:16:46 /bin/tar: yizaoyiwan: file changed as we read it

Show this error and no backup file created in target folder.

---

- gobackup version 0.5.0
- config file same to #7

**Comment by huacnlee** [comment]
fixed in 0.5.1

**Comment by huacnlee** [comment]
找对原因了，因为压缩的动作，在 0.5.0 版本里面错误的把压缩文件放在了被压缩的目录里面，导致一个引用冲突。

**Comment by huacnlee** [comment]
Fixed in 0.5.3
